solutions/fully-configurable/variables.tf

```diff
@@ -476,3 +476,43 @@ variable "cbr_rules" {
   description = "The list of context-based restriction rules to create. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/blob/main/solutions/fully-configurable/DA_docs.md#options-with-cbr)"
   default     = []
 }
+
+##############################################################
+# Ingress Secrets Manager Integration
+##############################################################
+
+variable "enable_secrets_manager_integration" {
+  type        = bool
+  description = "Integrate with IBM Cloud Secrets Manager so you can centrally manage Ingress subdomain certificates and other secrets. [Learn more](https://cloud.ibm.com/docs/containers?topic=containers-secrets-mgr)"
+  default     = false
+  nullable    = false
+  validation {
+    condition = anytrue([
+      !var.enable_secrets_manager_integration,
+      var.existing_secrets_manager_instance_crn != null
+    ])
+    error_message = "'existing_secrets_manager_instance_crn' should be provided if setting 'enable_secrets_manager_for_ingress' to true."
+  }
+}
+
+variable "existing_secrets_manager_instance_crn" {
+  type        = string
+  description = "CRN of the Secrets Manager instance where Ingress certificate secrets are stored."
+  default     = null
+}
+
+variable "secrets_manager_secret_group_id" {
+  type        = string
+  description = "Secret group ID where Ingress secrets are stored in the Secrets Manager instance. If not specified, the default group is used."
```

Comment:
This description is not correct. The DA will create a new secret group with the same name as the cluster if no value is passed here.  You should also mention it only applies if `enable_secrets_manager_integration` is true.